src/components/Dashboard/TodoList.vue

```diff
@@ -886,6 +911,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
       await AuthServices.fetchAffiliationInvitations(this.getAuthApiUrl, this.getIdentifier)
         .catch((err) => {
           console.log('Error fetching affiliation invitations for todo', err) // eslint-disable-line no-console
+          this.fetchAffiliationInvitationsErrorDialog = true
```

Comment:
Do you ever have to reset this to false? (Example: see resetCancelPaymentErrors().)

Or does this dialog just display until the user closes it?

Reply:
I do this on line ~13 `@okay="fetchAffiliationInvitationsErrorDialog=false"`

There is no cleanup actions other then closing the dialog, so wanted to avoid adding unnecessary method. If need for method arises, it can be added at that time.